### PR TITLE
NSE: Docs support

### PR DESCRIPTION
**Pull request by gokmen**
ps. this also sets new stack editor as default editor for stack templates. there is still option for using the old one.

### CODE REVIEW

File: client/new-stack-editor/lib/views/index.coffee [hunk 1]
```diff
@@ -156,13 +158,14 @@ module.exports = class StackEditor extends kd.View
       when Events.Menu.MakeTeamDefault
         computeController.makeTeamDefault { template: @getData() }
       when Events.Menu.Clone
-        @toolbar.setBanner {
+        @toolbar.setBanner
           message  : 'Clonning...'
```
Comment (sinan) [inline client/new-stack-editor/lib/views/index.coffee]: `Cloning…`

File: client/new-stack-editor/lib/views/sideview.coffee [hunk 2]
```diff
@@ -50,17 +52,43 @@ module.exports = class SideView extends BaseView
...
+        else
+          @title.hide()
         @setClass item.cssClass
         item.view.show()
 
       else
         @unsetClass item.cssClass
         item.view.hide()
 
+    { expanded, collapsed = yes } = options
```
Comment (sinan) [inline client/new-stack-editor/lib/views/sideview.coffee]: show as is (with the prev state) `unless options`
show expanded `if options.expanded`
show collapsed `unless options.expanded`

**Review by sinan — CHANGES_REQUESTED** [review]
👍

**Comment by gokmen** [comment]
@sinan all addressed and updated.

**Review by sinan — APPROVED** [review]

**Pull request merged**